orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/manageMembers/manageMembersFindClient.component.ts

```diff
@@ -128,13 +128,25 @@ export class ManageMembersFindClientComponent
       });
     });
 
-    this.modalService.notifyObservable$.subscribe(data => {
+    this.modalSubscription = this.modalService.notifyObservable$.subscribe(data => {
       if (
         data &&
         data.moduleId === "modalFindMemberConfirm" &&
         data.action === "close" &&
         data.input === "update"
       ) {
+        
+        if (data.object.redirectUris) {
+          data.object.redirectUris.forEach(element => {
+            if (element.actType && !element.actType.value) {
+              element.actType.value = "\"\""
+            }
+            if (element.geoArea && !element.geoArea.value) {
+              element.geoArea.value = "\"\""
+            }
```

Comment:
@lizkrznarich This was particularly hard to find. The backend needs quotes on empty strings, and this problem only happens if multiple URL of the same type are been added :mindblow:.
I am temporally fixing this on the front end, but maybe we can solve this on the backend later.

Reply:
@leomendoza123 Agree that we need to fix this up. It's a json field in the DB, but the data there is not consistent (values for default redirects vary between null, "", {"import-works-wizard":[]}, and{"import-works-wizard":[]} and {"import-works-wizard":["Global"]}. Will file some cards to clean up existing data and make sure geo are and act type are added consistently in future.